Gems/ROS2/Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -64,147 +64,6 @@ namespace ROS2
         }
     }
 
-    void CollidersMaker::BuildColliders(const sdf::Link* link)
```

Comment:
For other reviewers, the removed code a was dead code. Now creating colliders' product assets is done in "    CreateSceneManifest(sourceAssetPath,  assetInfoFile,  collider, visual)"